packages/amplify-graphql-api-construct-tests/src/__tests__/backends/configurable-stack/app.ts

```diff
@@ -76,7 +76,7 @@ new AmplifyGraphqlApi(stack, `${PREFIX}Api`, {
   authorizationModes: {
     defaultAuthorizationMode: 'AMAZON_COGNITO_USER_POOLS',
     userPoolConfig: { userPool },
-    apiKeyConfig: { expires: Duration.days(7) },
+    apiKeyConfig: { expires: Duration.days(2) },
```

Comment:
Unexpectedly, a Duration of 1 fails to deploy. Rather than spend a bunch of time troubleshooting, I went with 2, which is still less than 7. :)